config/initializers/mongo.rb

```diff
@@ -0,0 +1,17 @@
+module QME
+  patient_cache = Mongoid.default_client['patient_cache']
+  # create a unique index for patient cache, this prevents a race condition where the same patient can be entered multiple times for a patient
+  patient_cache.indexes.create_one({ 'value.measure_id' => 1, 'value.sub_id' => 1, 'value.effective_date' => 1, 'value.patient_id' => 1 },
```

Comment:
I can't find any information on `create_one` in Mongoid's documentation or on rubydoc.info; what's the difference between it and `create`? @rdingwell may be a better person to ask